### ISSUE TITLE
Firewall variables defined in output.tf but not in variables.tf

### ISSUE THREAD

**Issue by dalekurt** [hashicorp/terraform-google-consul#7]
Terraform version: 0.11.4

After executing `terraform plan` I get the following Errors. Apparently, there are not variables blocks for the follow though it is defined with the output. Is this an issue, or is it me?

```
Error: Error running plan: 4 error(s) occurred:

* module.consul_clients.output.firewall_rule_inbound_dns_url: Resource 'google_compute_firewall.allow_inbound_dns' not found for variable 'google_compute_firewall.allow_inbound_dns.self_link'
* module.consul_clients.output.firewall_rule_inbound_dns_name: Resource 'google_compute_firewall.allow_inbound_dns' not found for variable 'google_compute_firewall.allow_inbound_dns.name'
* module.consul_clients.output.firewall_rule_inbound_http_url: Resource 'google_compute_firewall.allow_inboud_http_api' not found for variable 'google_compute_firewall.allow_inboud_http_api.self_link'
* module.consul_clients.output.firewall_rule_inbound_http_name: Resource 'google_compute_firewall.allow_inboud_http_api' not found for variable 'google_compute_firewall.allow_inboud_http_api.name'
```

**Comment by blaisep** [comment]
@dalekurt I'm getting this also using Terraform 0.11.5
It prevents me from building a state file :-(
I think the problem may be in the google provider module or some other run time import because the string *`inboud`* does not appear anywhere in the in the project. Maybe @josh-padnick can point us in the right direction.

**Comment by dalekurt** [comment]
@blaisep Use version 0.10.8 for terraform you will have a better result.

**Comment by blaisep** [comment]
@dalekurt you have saved my b...acon. PM me a mailing address and I will send you some of my wife's cookies!

**Comment by josh-padnick** [comment]
This issue related to the breaking change in Terraform 11 around outputs was resolved in #15.
